lib/Utils.php

```diff
@@ -29,6 +29,18 @@ public static function runAuthProcFilter($nestedClass, array $config, &$state, $
         $authFilter->process($state);
     }
 
+    /**
+     * Check if the MFA auth proc filters (which were run) finished successfully. If everything is configured correctly,
+     * this should not throw an exception.
```

Comment:
```suggestion
     * Check whether MFA was performed, either locally (by running MFA auth proc filters) or at the upstream IdP.
```
Please update this docstring, it seems out of date.